syntax/grammar.mjs

```diff
@@ -390,15 +390,13 @@ let Identifier =
 /* -------------------------------------------------------------------------- */
 /* Content Characters
  *
- * Translation content can be written using most Unicode characters, with the
- * exception of C0 control characters (but allowing tab), surrogate blocks and
- * non-characters (U+FFFE, U+FFFF).
+ * Translation content can be written using any Unicode characters. However,
+ * some characters are considered special depending on the type of content
+ * they're in. See text_char and quoted_char for more information.
```

Comment:
Add a reference to recommendations.md here, too? Similar to how we link to valid.md in this file?